### PR TITLE
Bugfix InvalidParameterValueException is given when createFunction

### PR DESCRIPTION
**Pull request by abetomo**
In updateFunctionConfiguration, we set empty characters so that setting
can be erased, but failed when createFunction.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -114,7 +114,9 @@ Lambda.prototype._params = function (program, buffer) {
       Variables: {}
     },
     DeadLetterConfig: {
-      TargetArn: ''
+      // Set default value to null.
+      // Because createFunction fails if it is empty character.
+      TargetArn: null
     }
```
Comment (DeviaVir) [inline lib/main.js]: It might make more sense to leave it out completely and only add it when passed through to us? This might also solve for your TODO.

Comment (abetomo) [inline lib/main.js]: I tried implementing `undefined` check, but how about it?

Comment (DeviaVir) [inline lib/main.js]: @abetomo are you sure this solves for implicitly unsetting the DLQ?

Comment (abetomo) [inline lib/main.js]: @DeviaVir 
By writing `AWS_DLQ_TARGET_ARN=` in `.env`, empty characters will pass and you can cancel the setting.

**Pull request merged**